content/webapp/services/prismic/series.ts

```diff
@@ -0,0 +1,29 @@
+// Annoyingly this file and type is called series, but it is only used for articles
+import {
+  GroupField,
+  RelationField,
+  RichTextField,
+  TimestampField,
+  PrismicDocument,
+  SelectField,
+} from '@prismicio/types';
+import { CommonPrismicData } from './types';
+
+export type SeriesPrismicDocument = PrismicDocument<
+  {
+    color: SelectField<'teal' | 'red' | 'green' | 'purple'>;
```

Comment:
No turquoise?

Reply:
Nope, they've not had that choice previously. I'll tidy this up in my next PR.